src/exampleExplorer.ts

```diff
@@ -273,7 +273,8 @@ export class ExampleExplorer {
     const boardList = context.asAbsolutePath(
       path.join(FileNames.resourcesFolderName, FileNames.templatesFolderName, FileNames.boardListFileName)
     );
-    const boardsJson: { boards: Board[] } = require(boardList);
+    //const boardsJson: { boards: Board[] } = require(boardList);

```

Comment:
Can you help check with Kai/Di if we still need example Explorer in Workbench? I remember Kai has removed the command palette of example

Reply:
I think we still need the example explorer, we have a command for opening sample page and let user download sample code. @JerryYangKai @dilin-MS can you confirm?

Reply:
We still have the example Explorer. What you mention may be the `Azure IoT Device Workbench: Help` command, which will open a welcome page showing a oil pumping machine in the past and the page was later deleted by @JerryYangKai . Now the command will lead to a external URL.